Makefile

```diff
@@ -30,27 +31,27 @@ lint: \
 
 lint-black: ## lint back-end python sources with black
 	@echo 'lint:black started…'
-	@$(COMPOSE_TEST_RUN_APP) black src/ashley sandbox tests
+	@$(COMPOSE_TEST_RUN_APP_NODEPS) black src sandbox tests
```

Comment:
If you move dockerize away from the base image, this commit can be removed IMO

Reply:
In fact, this problem was not related to dockerize but docker-compose (because of the `depends_on = "postgresql"`).

I though it was slower without `--no-depts` option, because before running each linter, docker-compose displayed : `Starting ashley_postgresql_1 ... done`

But in fact, after verification, there is no difference in execution time.
So i'll follow your advice and remove this commit :wink: